### ISSUE TITLE
Where should accessibility/DEI statement go? 

### ISSUE THREAD

**Issue by maneesha** [carpentries/new-website#183]
On the new site, Kari added an [accessibility statement on our About page](https://carp-new-website.netlify.app/about-us/#diversity-equity-and-inclusion-statement).

This does not have as muc information as our current  [Diversity, Equity, Inclusion, and Accessibility page](https://carpentries.org/eia/).  What additional information should be added to the accessibility section on our new site?

**Comment by OscarSiba** [comment]
The section on "Request an Accommodation" and "Resources" that are currently on our [Diversity, Equity, Inclusion, and Accessibility page](https://carpentries.org/eia/) page should go under the Workshops section of the new website. 

They should come just before the Contact Us section of that page: https://carp-new-website.netlify.app/workshops/

The writeup under the Request an Accommodation section should be modified to read as follows:

"At The Carpentries, we are committed to building a diverse, equitable, and inclusive community that values all individuals and their unique identities. We prioritise accessibility and inclusivity in our curricula and programs and value transparency, fairness, and honesty to build trust within our community. Building an inclusive community is an ongoing process that requires consistent effort and commitment, and we strive for continuous improvement.

To request an accommodation for a Carpentries event, please fill out the [accommodation request form](https://carpentries.typeform.com/to/B2OSYaD0).

If you have questions or need assistance with the accommodation form please [email us](mailto:team@carpentries.org)."

@sheraaronhurt can you please also confirm whether you are happy with this suggested change.

**Comment by maneesha** [comment]
@OscarSiba Can you share why it goes in the workshops section? People should have opportunities to request accommodations for any event type (not just workshops), correct?

**Comment by OscarSiba** [comment]
You are correct @maneesha, and in that case I would also recommend that the "Request an Accommodation" section is replicated under the [Community Events](https://carp-new-website.netlify.app/community/events/) and/or Get Involved page. Essentially, the accommodation request form should be placed in a context where someone requiring an accommodation can immediately find it. And this is when they are planning to attend a workshop or a community event or to engage with the community in some way. I do not believe the "About" page is not such a place.

**Comment by maneesha** [comment]
@OscarSiba At your suggestion, I have added the entire accessibility section to the [Workshops page](https://deploy-preview-206--carp-new-website.netlify.app/workshops/), the [Community Events page](https://deploy-preview-206--carp-new-website.netlify.app/community/events/), and the [Get Involved page](https://deploy-preview-206--carp-new-website.netlify.app/community/get-involved/) (PR previews hyperlinked here, not the test deployed site).  I still think that having it in its own section that all other pages link to would make more sense. In this preview, we have added it in full to three sections, but not to CLDT, or Instructor Training, or other places where it may be relevant.

**Comment by sheraaronhurt** [comment]
I would recommend having a dedicated page that we then link to. It was my understanding that for something like this, we would want to link it out so that we don't have to change it on multiple pages.

**Comment by maneesha** [comment]
I discussed this with @OscarSiba. We agreed the text on the "About" page should be expanded, and then this section should be linked to from every relevant page.  I can make the change on the "About" page.  

@OscarSiba  will list where we should link to this page from.

**Comment by maneesha** [comment]
@OscarSiba the statement now appears at the bottom of the [about page](https://carp-new-website.netlify.app/about-us/#diversity-equity-and-inclusion-statement).

Can you share here where we should have links to this section?

**Comment by OscarSiba** [comment]
Please have the following statement as a subheading on the following pages:

##Request an accommodation 
The Carpentries is committed to providing inclusive and accessible content that enables all individuals, including those with disabilities, to participate and engage fully. To request an accommodation for a Carpentries event, please visit our [Diversity, Equity, and Inclusion Statement](https://deploy-preview-206--carp-new-website.netlify.app/about-us/#diversity-equity-and-inclusion-statement) and fill out the accommodation request form provided.

- [ ] Before the Contact us section of the Workshops page: https://deploy-preview-206--carp-new-website.netlify.app/workshops/
- [ ] Below "Search Lessons" on this page: https://deploy-preview-206--carp-new-website.netlify.app/lessons/
- [ ] Before the Contact us section of the Instructor Training page: https://deploy-preview-206--carp-new-website.netlify.app/instructor-training/
- [ ] Before the Contact us section of the Trainer Training page: https://deploy-preview-206--carp-new-website.netlify.app/trainer-training/
- [ ] Before the Contact us section of the Lesson Development page: https://deploy-preview-206--carp-new-website.netlify.app/lesson-development/
- [ ] Below the brief introduction section on the Community Events page: https://deploy-preview-206--carp-new-website.netlify.app/community/events/
- [ ] As the last section in the Get Involved page: https://carp-new-website.netlify.app/community/get-involved/
